checkstyle/suppressions.xml

```diff
@@ -32,6 +32,8 @@
     <suppress checks="ParameterNumber"
               files="Fetcher.java"/>
     <suppress checks="ParameterNumber"
+              files="Sender.java"/>
```

Comment:
Is this okay? I added a new parameter to the Sender class, which pushed it over the checkstyle limit.